scripts/docker-integration-tests/aggregator/m3aggregator.yml

```diff
@@ -121,6 +121,7 @@ aggregator:
   bufferDurationBeforeShardCutover: 10m
   bufferDurationAfterShardCutoff: 10m
   bufferDurationForFutureTimedMetric: 10m # Allow test to write into future.
+  bufferDurationForPastTimedMetric: 10s # Don't wait too long for timed metrics to flush.
```

Comment:
now that the integration tests correctly use a timed list instead of a standard list, the test would timeout waiting for the default buffer past of 60s.